src/main/scala/esmeta/analyzer/TypeAnalyzer.scala

```diff
@@ -287,6 +289,7 @@ class TypeAnalyzer(
       data = Yaml(
         "duration" -> f"${sem.elapsedTime}%,d ms",
         "error" -> errors.size,
+        "unreachable" -> unreachables.size,
```

Comment:
Dumping unreachable nodes and logging the statistics should be different; the summary should contain the whole number of nodes the analysis doesn't touch since we should check that the analysis is more/less sound after the patch applies.

Reply:
In current implementation, the `unreachable` means all of node points that have no corresponding state in `semantics.npMap`.